README.md

```diff
@@ -1,8 +1,153 @@
 <!-- @format -->
 
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with
-[Create React App](https://github.com/facebook/create-react-app).
+# Project
 
-## Available Scripts
+> This Webpage is for a calculator app called Math
+> Magicians bad it was built using React!
+
+![screenshot]()
+
+## Built With
+
+- HTML / CSS / JavaScript / React
+
+## Live Demo
+
+- [Live Demo Link](https://zhadier.github.io/)
+- [Video Demo Link]()
```

Comment:
- [ ] the live demo en video demo link are both not working, please fix or remove them from the readme file.